ui/src/apps/clusterInfo/RootComponent.js

```diff
@@ -36,7 +36,7 @@ const App = () => {
                   <ComponentPanel field="pd" data={cluster} />
                 </Col>
               </Row>
-              <ClusterInfoTable cluster={cluster} />
+              <ClusterInfoTable cluster={{cluster, setCluster}} />
```

Comment:
It is better to separate the props likes `<ClusterInfoTable cluster={cluster} onChange={setCluster} />`